smartmeter/smartmeter.ino

```diff
@@ -105,7 +117,7 @@ void setup()
     Display::self().setData(Data::self().data);
     Connection::self();
     Connection::self().setData(Data::self().data);
-    Connection::self().addWifi("SSID", "pass");
+    Connection::self().addWifi("Gabz & Rafa", "gabzgabz");
```

Comment:
Okey, no everybody know your wifi password.

Reply:
When adding changes, use `git add -p`, this will help you.

Reply:
Good to know!